LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/LoRaDevice.cs

```diff
@@ -261,7 +268,86 @@ public async Task<bool> InitializeAsync()
             return false;
         }
 
-        int GetTwinPropertyIntValue(dynamic value)
+        private void InitializeFrameCounters(Twin twin)
+        {
+            var toReport = new TwinCollection();
+
+            bool reset = false;
+            // check if there is a reset we need to process
+            if (twin.Properties.Desired.Contains(TwinProperty.FCntResetCounter))
+            {
+                var resetDesired = GetTwinPropertyIntValue(twin.Properties.Desired[TwinProperty.FCntResetCounter].Value);
```

Comment:
Just want to make sure I understand. To reset the frame counter one should:
- Add `FCntResetCounter` to desired property. If the value already exists, increment by one
- Set the initial value in desired `FcntUp` and `FcntDown`

Reply:
Yes.
The first time you do not need to touch the ResetCounter, as reported will be empty, and we will use the specified start values, if they are there in the desired collections.